src/main/java/uk/gov/hmcts/reform/sscs/service/NotificationService.java

```diff
@@ -71,7 +92,11 @@ public void manageNotificationAndSubscription(NotificationWrapper notificationWr
 
         log.info("Notification event triggered {} for case id {}", notificationType.getId(), caseId);
 
-        if (notificationWrapper.getNotificationType().isAllowOutOfHours() || !outOfHoursCalculator.isItOutOfHours()) {
+        if (notificationWrapper.getNotificationType().isToBeDelayed()
```

Comment:
I think possibly this should go after the isAllowOutOfHours block below. It probably doesn't matter for appealCreated notifications as they are not affected by the allow out of hours logic. However, if we introduce another notification that we want to delay for 300 seconds and the allowOutOfHours flag is false, when it is out of hours we could end up sending this notification after 300 seconds which is not correct. Let me know what you think?